apps/s_client.c

```diff
@@ -2105,6 +2107,32 @@ int s_client_main(int argc, char **argv)
                 goto shut;
         }
         break;
+    case PROTO_LDAP:
+        {
+            char *ldap_tls_genconf = "asn1=SEQUENCE:LDAPMessage\n"
+                "[LDAPMessage]\n"
+                "messageID=INTEGER:1\n"
+                "extendedReq=EXPLICIT:23A,IMPLICIT:0C,FORMAT:ASCII,OCT:1.3.6.1.4.1.1466.20037\n";
+            long errline;
+            char *genstr;
```

Comment:
if genstr and atyp are moved later, then they can be init'd as part of their declaration.